### ISSUE TITLE
Other theme issues

### ISSUE THREAD

**Issue by Vladimir-csp** [johanmalm/jgmenu#33]
There is 1 pixel transparent line around menu http://i.imgur.com/789WESy.png (visible because of compton's shadow). Tint2's taskbar background has lighter color 1 pixel border, but menu's background is solid and without border.
Maye add generic `menu_padding` setting (with lower priority than `menu_padding_*`)?
If tint2conf parsing is enabled, consider taking values for `menu_padding_*` and `item_margin_y` from `taskbar_padding`?
10 default for `menu_padding_*` is questionable, maybe 5 would be better?

**Comment by johanmalm** [comment]
I think I've fixed the window border issue. Would appreciate if you could test it too.

Grateful for other feedback too. Good thoughts.

**Comment by Vladimir-csp** [comment]
Yes, the transparent line is gone.

**Comment by johanmalm** [comment]
I have implemented your suggestion wrt taking values for menu_padding_* and item_margin_y from tint2's taskbar_padding. This could be implemented in several ways - hopefully I've taken a fairly 'safe' approach. See set_padding_and_item_margin() in t2conf.c if you want to see the details. I'm of a mind that there are so many permutations, that if a user really want to fine-tune, they will have to be explicit in their jgmenurc (using tint2_rules=0 and comment out all options they want to be iaw tint2rc).

I've changed the default padding from 10 to 5 as you suggested. Yes, I probably got a bit carried away :-)

I prefer not to include a generic menu_padding setting. Over the last month I've overhauled and slimmed down on config options. I still feel we're at the "let's not grow arms and legs quite yet" stage.
